packages/snaps-controllers/src/snaps/SnapController.test.ts

```diff
@@ -1162,24 +1110,21 @@ describe('SnapController', () => {
       },
     });
 
-    const timeoutPromise = new Promise<boolean>((resolve) => {
-      setTimeout(() => {
-        resolve(true);
-      }, 200);
-    });
+    const timeoutPromise = sleep(200);
 
     expect(
       // Race the promises to check that handlerPromise does not time out
       await Promise.race([handlerPromise, timeoutPromise]),
-    ).toBe(true);
+    ).toBe('test1');
```

Comment:
Where does `test1` come from here?

Reply:
From `DEFAULT_SNAP_BUNDLE`. It returns `method + id`.